### ISSUE TITLE
Multiple ASNs for single ISP/Transit provider

### ISSUE THREAD

**Issue by schadom** [cloudflare/isbgpsafeyet.com#177]
Many large ISPs and Transit providers operate multiple ASNs. The list should allow for multiple ASNs under a single entity/organization.

**Comment by pshem** [comment]
That would break the ordering based on ASN rank in `operators.csv` and on ASN number in `twitter.csv` though

**Comment by jfuruness** [comment]
Agreed, multiple ASNs would be quite helpful. Not much point in just including one. This would be incredibly useful for research.
